health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -79,115 +78,107 @@ export const UICustomizations = {
-            optionKey={"name"}
-            selected={{ code: "1", name: "Actions" }}
-          />
-          // <p>$${value}</p>
-        );
-      }
-
-      if (key === "NAME_OF_MICROPLAN") {
-        if (value && value !== "NA") {
+      switch (key) {
+        case "ACTIONS":
+          let options = [];
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)